src/advanced/node-management.md

```diff
@@ -324,6 +324,26 @@ Response is a JSON object with the following fields:
 }
 ```
 
+### User agent info
+
+```None
+GET {system_base_path}/user_agent
+```
+
+#### Parameters
+
+None.
+
+#### Response
+
+Returns string containing information about Exonum, Rust and OS version.
+
+#### Response example
+
+```None
+"exonum 0.6.0/rustc 1.26.0-nightly (2789b067d 2018-03-06)\n\n/Mac OS10.13.3"
```

Comment:
Off-topic, but I can't help myself: why isn't the output more structured, like

```json
{
  "exonum": "0.6.0",
  "rustc": { "version": "1.26.0-nightly", "commit": "2789b067d", "date": "2018-03-06" },
  "os": { "type": "Mac OS", "version": "10.13.3" }
}
```

Reply:
I'm not sure that we are ready for a more strict format.